### ISSUE TITLE
Shape Painter does not work on mobile devices if using high quality anti-aliasing

### ISSUE THREAD

**Issue by fizzingpop** [4ian/GDevelop#6286]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/4ian/GDevelop/issues)

### Describe the bug

Shape painter objects will not show on Android devices when exported if you select high-quality anti-aliasing (and possibly medium). However low-quality and no anti-aliasing still work.

### Steps to reproduce

1. Use a shape painter
2. Set to high-quality anti-aliasing in the object
3. Export to Android. It will not show up.

### GDevelop platform

Desktop

### GDevelop version

5.3.188

### Platform info

<details>

Developed on Windows 11
Exported to Android
</details>

**Comment by Oxey405** [comment]
Oh... that's probably something of my doing i'll look into it !

**Comment by Oxey405** [comment]
Could you provide some details about the phone you tested this on ? Is it an old phone (Like 4 years) ?
Because it could very well be that your phone is just too old to support such levels of Anti-aliasing.
It could also be a problem with the underlying rendering engine PixiJS. And if it's a problem with PixiJS, it's not something fixable in GDevelop code...

**Comment by fizzingpop** [comment]
> Could you provide some details about the phone you tested this on ? Is it an old phone (Like 4 years) ? Because it could very well be that your phone is just too old to support such levels of Anti-aliasing. It could also be a problem with the underlying rendering engine PixiJS. And if it's a problem with PixiJS, it's not something fixable in GDevelop code...

Galaxy A34 running Android 14 so not an old phone at all.

**Comment by Oxey405** [comment]
Mhh weird... Can you try using other Anti-aliasing settings ?

**Comment by fizzingpop** [comment]
> Mhh weird... Can you try using other Anti-aliasing settings ?

Yes low and medium work fine. Only high does not, it makes the shape painter not appear at all.

**Comment by Oxey405** [comment]
> > Mhh weird... Can you try using other Anti-aliasing settings ?
> 
> Yes low and medium work fine. Only high does not, it makes the shape painter not appear at all.

Ok this really looks like a renderer issue because the code for high and any other is literally the same...
